x-pack/plugin/esql/qa/testFixtures/src/main/resources/chunk.csv-spec

```diff
@@ -3,29 +3,48 @@
 #
 
 chunkExample
```

Comment:
do we have any tests when we pass NULL or a multi value keyword column to `chunk`?

Reply:
Multi valued already exists [here](https://github.com/elastic/elasticsearch/blob/e8a9eb1703eaa4fa619f280f6a67007fa5335bef/x-pack/plugin/esql/qa/testFixtures/src/main/resources/chunk.csv-spec#L111) but will add one for `null`